src/derivkit/forecasting/integrations/getdist_dali_samples.py

```diff
@@ -0,0 +1,312 @@
+
+    The target posterior is evaluated with
+    :meth:`derivkit.forecasting.expansions.logposterior_dali`. Samples are drawn from
+    a Fisher–Gaussian kernel centered on ``theta0`` and reweighted by the difference
+    between the target log-posterior and the kernel log-density.
+
+    Notes:
+        ``MCSamples.loglikes`` follows GetDist convention and stores ``-log(posterior)``
+        up to an additive constant.
+
+    Args:
+        theta0: Fiducial parameter vector with shape ``(p,)`` for ``p`` parameters.
+        fisher: Fisher matrix at ``theta0`` with shape ``(p, p)``.
+        g_tensor: DALI third-order tensor ``G`` with shape ``(p, p, p)``.
+        h_tensor: Optional DALI fourth-order tensor ``H`` with shape ``(p, p, p, p)``.
+        names: GetDist parameter names (length ``p``).
+        labels: GetDist parameter labels (length ``p``).
+        n_samples: Number of kernel samples to draw.
+        kernel_scale: Scale factor applied to the Fisher covariance for the kernel.
+        convention: DALI convention passed through to ``logposterior_dali``.
+        seed: Random seed for kernel sampling.
+        prior_terms: Prior terms used to build a prior via ``build_prior`` (ignored if
+            ``logprior`` is provided).
+        prior_bounds: Bounds used to build a prior via ``build_prior`` (ignored if
+            ``logprior`` is provided).
+        logprior: Optional custom log-prior ``logprior(theta)``.
+        hard_bounds: Optional hard support bounds.
+        label: Label attached to the returned ``MCSamples``.
+
+    Returns:
+        GetDist ``MCSamples`` with importance ``weights`` and GetDist-style ``loglikes``.
+
+    Raises:
+        ValueError: If shapes are inconsistent or mutually exclusive options are provided.
+        RuntimeError: If all samples are rejected by bounds or prior support.
+    """
+    fiducial = np.asarray(theta0, dtype=float)
+    fisher_matrix = np.asarray(fisher, dtype=float)
+    dali_g = np.asarray(g_tensor, dtype=float)
+    dali_h = None if h_tensor is None else np.asarray(h_tensor, dtype=float)
+    validate_dali_shapes(fiducial, fisher_matrix, dali_g, dali_h)
+
+    n_params = int(fiducial.size)
+    if len(names) != n_params or len(labels) != n_params:
+        raise ValueError("names/labels must match number of parameters")
+
+    if logprior is not None and (prior_terms is not None or prior_bounds is not None):
+        raise ValueError("Use either `logprior` or (`prior_terms`/`prior_bounds`), not both.")
+
+    if hard_bounds is not None and (prior_terms is not None or prior_bounds is not None):
+        raise ValueError(
+            "Ambiguous support: you passed `hard_bounds` and also `prior_bounds`/`prior_terms`.\n"
+            "Choose ONE support mechanism:\n"
+            "  - use `hard_bounds` (set prior_bounds/prior_terms to None), OR\n"
+            "  - encode support via the prior (set `hard_bounds=None`)."
```

Comment:
```suggestion
            "Ambiguous support: both `hard_bounds` and `prior_bounds`/`prior_terms` were supplied.\n"
            "Choose ONE support mechanism:\n"
            "  - use `hard_bounds` (set prior_bounds/prior_terms to None), OR\n"
            "  - encode support via the prior (set `hard_bounds=None`)."
```
I'm not sure if the backticks add much here. A statement like `prior_bounds/prior_terms` is ambiguous.